### PR TITLE
Fix message popup

### PR DESCRIPTION
**Pull request by fortmarek**
## Description of the pull request

- fix views having bigger height than they should -> I am not sure why [this](https://github.com/GetStream/stream-chat-swift/compare/bugfix/reactions?expand=1#diff-a41621945186dd3f456888db631f0bdc157e35fedb332477c9f2d9790e922d17L200) was there before, but the behavior was different from `UIStackView` which I found weird in this case. 
- fix not tappable reactions -> gesture recognizer added to `scrollView` was being triggered instead of the reactions

### CODE REVIEW

File: Sources/StreamChatUI/CommonViews/ContainerStackView.swift [hunk 1]
```diff
@@ -195,14 +195,14 @@ public class ContainerStackView: UIView {
...
                 customConstraints.append(
-                    lView.widthAnchor.constraint(equalTo: rView.widthAnchor).priority(distribution == .equal ? .required : .lowest)
+                    lView.widthAnchor.constraint(equalTo: rView.widthAnchor)
                 )
             } else {
                 customConstraints.append(
                     lView.heightAnchor.constraint(equalTo: rView.heightAnchor)
-                        .priority(distribution == .equal ? .required : .lowest)
                 )
             }
```
Comment (VojtaStavik) [inline Sources/StreamChatUI/CommonViews/ContainerStackView.swift:207]: cc @nuno-vieira 😅

File: Sources/StreamChatUI/CommonViews/ContainerStackView.swift [hunk 2]
```diff
@@ -195,14 +195,14 @@ public class ContainerStackView: UIView {
 
         // Add constraints for distribution
         zip(subviews, subviews.dropFirst()).forEach { lView, rView in
+            guard distribution == .equal else { return }
```
Comment (VojtaStavik) [inline Sources/StreamChatUI/CommonViews/ContainerStackView.swift:198]: we can probably move this `guard` outside the loop ;-)

**Review by VojtaStavik — COMMENTED** [review]
So what was the issue?

**Comment by fortmarek** [comment]
> So what was the issue?

The issue with the height of actions was that they "tried" to be as big as the message (the preceding view). Even with very low priority it influenced the view (one could most likely fix it with `setContentHugging` but I don't it'd be a proper solution here)

File: Sources/StreamChatUI/CommonViews/ContainerStackView.swift (hunk 2 above)
Comment (nuno-vieira) [reply Sources/StreamChatUI/CommonViews/ContainerStackView.swift:198]:
```
if distribution == .equal {
            zip(subviews, subviews.dropFirst()).forEach { lView, rView in
                if axis == .horizontal {
                    customConstraints.append(
                        lView.widthAnchor.constraint(equalTo: rView.widthAnchor)
                    )
                } else {
                    customConstraints.append(
                        lView.heightAnchor.constraint(equalTo: rView.heightAnchor)
                    )
                }
            }
        }
```

Comment (fortmarek) [reply Sources/StreamChatUI/CommonViews/ContainerStackView.swift]: yeah, I have just done that 👍

**Review by VojtaStavik — APPROVED** [review]

**Comment by VojtaStavik** [comment]
@fortmarek do you have time to fix the failing test and merge today?

**Pull request merged**